internal/service/uow/exec_test.go

```
@@ -1,131 +1,293 @@
+	"db-worker/internal/storage/mocks"
+	"db-worker/internal/storage/testtransaction"
 	"errors"
+	"reflect"
 	"testing"
 
+	"github.com/golang/mock/gomock"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 //nolint:funlen,dupl // много тест-кейсов, одинаковые тест-кейсы для разных тестов
```

golangci-lint (2.5.0)

[error] directive `//nolint:funlen,dupl // много тест-кейсов, одинаковые тест-кейсы для разных тестов` is unused for linter "dupl"

(nolintlint)